test/steps/ffmpeg.go

```diff
@@ -80,8 +80,11 @@ func (s *StepContext) TheSourceManifestIsWrittenToStorageWithinSeconds(secs, num
 	if mediaManifest.Version() != 3 {
 		return fmt.Errorf("expected manifest to be HLSv3 but got version: %d", mediaManifest.Version())
 	}
-	if mediaManifest.TargetDuration != 10.0 {
-		return fmt.Errorf("expected manifest to have a Target Duration of 10 but got: %f", mediaManifest.TargetDuration)
+	if mediaManifest.TargetDuration != 11.0 {
```

Comment:
What is this `11.0`?

Reply:
The different ffmpeg filter seems to have slightly different behaviour with estimating the target duration. We could override it to be our [requested duration](https://github.com/livepeer/catalyst-api/pull/909/files#diff-800ac29ee96767d39950a993dc844eac954d9d93ce5cd9973fbeadf5cc7f0530L14) but I'm not sure that it matters. cc @thomshutt

Reply:
I think that as long as it's larger than the largest segment it produces then that's fine

Reply:
Are the segments it produces the same size as before (other than the problematic one)?

Reply:
Mostly the same but small differences. e.g. for this test:
`-f hls`
```
2023/10/11 16:03:46 #EXTM3U
#EXT-X-VERSION:3
#EXT-X-TARGETDURATION:10
#EXT-X-MEDIA-SEQUENCE:0
#EXT-X-PLAYLIST-TYPE:VOD
#EXTINF:10.000000,
index0.ts
#EXTINF:10.000000,
index1.ts
#EXTINF:10.000000,
index2.ts
#EXTINF:0.033333,
index3.ts
#EXT-X-ENDLIST
```
`-f segment`
```
2023/10/11 16:02:58 #EXTM3U
#EXT-X-VERSION:3
#EXT-X-PLAYLIST-TYPE:VOD
#EXT-X-MEDIA-SEQUENCE:0
#EXT-X-TARGETDURATION:11
#EXTINF:10.044,
index0.ts
#EXTINF:10.000,
index1.ts
#EXTINF:10.000,
index2.ts
#EXTINF:0.033,
index3.ts
#EXT-X-ENDLIST